app/utils/estimate_prioritization/estimate_prioritization.py

```diff
@@ -0,0 +1,108 @@
+import pandas as pd
+from statsmodels.stats.proportion import proportion_confint
+from app.utils.estimate_prioritization import prioritization_criteria_testunadj, \
+    prioritization_criteria_testadj, pooling_function_maps
+
+
+def get_pooled_estimate(estimates):
+
+    if estimates.shape[0] == 1:
+        pooled = estimates.iloc[0]
+    else:
+        # use the estimate with the max denominator as the 'base' for the pooled estimate
+        pooled = estimates.loc[estimates['denominator_value'].idxmax(axis = 1)] 
+
+        # pool all values that can be pooled
+        for (summary_type, cols_to_summarize, summary_function) in pooling_function_maps:
+            if summary_function is not None:
+                valid_cols_to_summarize = set(cols_to_summarize) & set(estimates.columns)
+
+                for col in valid_cols_to_summarize:
+                    pooled.at[col] = summary_function(estimates, col)
+
+        # generate new values as necessary
+        pooled.at['numerator_value'] = int(pooled.at['serum_pos_prevalence'] * pooled.at['denominator_value'])
+        if 'estimate_name' in pooled.index:
+            pooled.at['estimate_name'] += '_pooled'
+        pooled.at['seroprev_95_ci_lower'], pooled.at['seroprev_95_ci_upper'] = proportion_confint(count = pooled.at['numerator_value'],
+                                                                                          nobs = pooled.at['denominator_value'],
+                                                                                          alpha = 0.05,
+                                                                                          method = 'jeffreys')
+
+        # TODO: once test adjustment is in, we'll need to add a parallel set of code to the above
```

Comment:
Making a commit to add the test adjustment pooling code here: ways to pool `[adj_prevalence, adj_sensitivity, adj_specificity, adj_prev_ci_lower, adj_prev_ci_upper, ind_eval_type]` Will wrap in try / except so we catch any keyerrors before merge or when the research table isn't included.